NeuroID/src/main/java/com/neuroid/tracker/utils/NIDTextWatcher.kt

```diff
@@ -48,32 +50,38 @@ class NIDTextWatcher(
             val pasteCount = pastedText.length
             if (sequence.toString().contains(pastedText) && (pasteCount == count)) {
                 // The change is likely due to a paste operation
-                val ts = System.currentTimeMillis()
-                val gyroData = NIDSensorHelper.getGyroscopeInfo()
-                val accelData = NIDSensorHelper.getAccelerometerInfo()
 
-                val metadataObj = JSONObject()
-                metadataObj.put("clipboardText", "S~C~~${pastedText.length}")
+                val currentPastedHashValue = sequence?.toString()?.getSHA256withSalt()?.take(8)
```

Comment:
Might not need to use the SHA256. String has a hashCode() that should suffice for this. Can consider updating next time.

Reply:
updated to use hashCode. Thanks!